wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -253,19 +253,52 @@ static function enqueue_scripts() {
 	}
 
 	/**
+	 * Send an error message and any additional data
+	 */
+	static function json_error( $message, $data = array() ) {
```

Comment:
This method expects `$message` to be set, but lines 115 and 121 invoke it without any arguments.